modules/admin_manual/pages/configuration/search/index.adoc

```diff
@@ -24,8 +25,7 @@ Versions 6 and 7 are yet to be supported.
 
 To install the search_elastic app:
 
-. Clone the {search_elastic-repo-url}[search_elastic repository] to the apps directory (or xref:installation/apps_management_installation.adoc#using-custom-app-directories[custom apps directory]) of your ownCloud instance.
-. On the command-line, from within the search_elastic app directory, install the app's dependencies, by running `composer install`.
+. Download and extract the tarball of {search_elastic-releases-url}[the latest release] of the search_elastic app to the apps directory (or xref:installation/apps_management_installation.adoc#using-custom-app-directories[custom apps directory]) of your ownCloud instance.
```

Comment:
Why not download from the marketplace?

Reply:
Because, as I raised previously (https://github.com/owncloud/docs/pull/1545#discussion_r308111592 & https://github.com/owncloud/docs/pull/1545#issuecomment-518526790), it's not available in the Marketplace yet. As soon as it is, the documentation can be updated accordingly.

Reply:
Yes, agreed. But this resource you are referring to is not public.

So we have two options:
- Point the user to the marketplace, (I know that the app is not yet there)
- Have no installation instructions, because the app is not yet available to the public.

Reply:
What URL it will be available under? I can use that in the expectation that it will be what goes live.

Reply:
Given that the app's not available, it would make most sense to leave it as it is, and update it once the live URL is known.

Reply:
When is https://github.com/owncloud/search_elastic expected to be made public?

Reply:
So the final url will be https://marketplace.owncloud.com/apps/search_elastic. Please add that, then approve and go :)

Reply:
> So the final url will be https://marketplace.owncloud.com/apps/search_elastic. Please add that, then approve and go :)

Done!